main.js

```diff
@@ -300,20 +302,18 @@ function setVideoElements() {
 
   const video = Videos.list[Videos.video];
 
-  document.getElementsByTagName("source")[0].src = "video/" + video.file;
+  document.getElementsByTagName("source")[0].src = "/video/" + video.file;
```

Comment:
This breaks relative paths in favor of absolute paths. I see no good reason to ruin subfolder install capabilities.